src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.spec.ts

```diff
@@ -108,7 +113,8 @@ describe('StripeUPEPaymentStrategy', () => {
             orderActionCreator,
             stripeScriptLoader,
             storeCreditActionCreator,
-            'en_US'
+            'en_US',
+            remoteCheckoutRequestSender
```

Comment:
```suggestion
            remoteCheckoutRequestSender,
```

Reply:
That's an unnecessary trailing comma, tslint doesn't like that

Reply:
@BC-PabloRivera  Got it wrong.🙏🏻 My bad.